app/controllers/reviews_controller.rb

```diff
@@ -161,6 +161,11 @@ def vote_down
       current_user.vote_against(@review)
     end
 
+    if ((@review.votes_against > -9) && @review.comment != "")
+      @review.deleted = true
+      @review.save
```

Comment:
You can probably group these two lines together. A quick search looks like `@review.update_attribute(:deleted, false)` might do the trick, but look into it.